editor/js/main.js

```diff
@@ -283,6 +283,7 @@
         if (RED.settings.theme('palette.editable') !== false) {
             RED.palette.editor.init();
         }
+        RED.gitconfig.init();
```

Comment:
This should be called within the `RED.projects.init` function - https://github.com/node-red/node-red/blob/1b632894d3d601d09585271267fc8405c3a436dc/editor/js/ui/projects.js#L955

That will allow us to avoid adding this settings dialog if the user has projects/version control disabled (something we haven't implemented yet).

That also allows you to pass in a reference to `projectsAPI` so you don't need to duplicate the `sendRequest` function.